books/free-programming-books-subjects.md

```diff
@@ -670,7 +670,6 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 
 * [A short introduction to operating systems (2001)](http://markburgess.org/os/os.pdf) - Mark Burgess (PDF)
 * [Computer Science from the Bottom Up](http://www.bottomupcs.com)
-* [How to Make a Computer Operating System](https://github.com/SamyPesse/How-to-Make-a-Computer-Operating-System) (:construction: *in process*)
```

Comment:
did you intend to remove this? if so, why?

Reply:
Oh, no I did not intent to remove that line sorry